tests/Controller/CRUDControllerTest.php

```diff
@@ -2662,6 +2662,28 @@ public function testAclActionAclNotEnabled(): void
         $this->controller->aclAction(null);
     }
 
+    /**
+     * @group legacy
```

Comment:
Can you add a next major comment about what to do ?

Reply:
done